### PR TITLE
Update kubecost-single-account-permissions.yaml

### PR DESCRIPTION
**Pull request by brstuder**
Changes requested by @chipzoller as part of https://github.com/kubecost/docs/issues/1029

### CODE REVIEW

**Review by chipzoller — APPROVED** [review]
Should also receive input from Solutions team.

**Comment by brstuder** [comment]
Asking Sean right now

**Review by srpomeroy — CHANGES_REQUESTED** [review]
Line 106 needs to be updated to reflect the new parameter name.

We will also need to update `kubecost-masterpayer-account-permissions.yaml`

**Comment by brstuder** [comment]
Good catch, both should be corrected now

File: kubecost-masterpayer-account-permissions.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ AWSTemplateFormatVersion: '2010-09-09'
 Description: Kubecost Masterpayer Account Template
 
 Parameters:
-  AthenaCURBucket:
+  S3CURBucket:
     Description: The bucket where the CUR is sent from the “Setting up the CUR” step.
```
Comment (srpomeroy) [inline kubecost-masterpayer-account-permissions.yaml]: Nit. Do we want to update this to the same as the other CFT?
```suggestion
    Description: The bucket created to receive CUR data upon CUR creation.
```

**Review by srpomeroy — APPROVED** [review]
Two nits.

Should we update the name of the file from `kubecost-masterpayer-account-permissions.yaml` to `kubecost-management-account-permissions.yaml`? There are obviously repercussions for doing this elsewhere (docs).

**Comment by brstuder** [comment]
I'm fine to update the file, as well as sweep for any other mentions of "masterpayer" in the docs which I believe AWS has retired.

**Comment by brstuder** [comment]
I've renamed the file name and updated the link in the AWS doc, but there are other mentions across the AWS doc of masterPayerARN as a value. Maybe we can deal with all those in a separate PR?

**Comment by srpomeroy** [comment]
> I've renamed the file name and updated the link in the AWS doc, but there are other mentions across the AWS doc of masterPayerARN as a value. Maybe we can deal with all those in a separate PR?

Yeah... Updating that property name is a larger lift as it's in the application itself.

**Pull request merged**